packages/client/src/index.ts

```diff
@@ -1,14 +1,67 @@
 import { Property, Signature } from '@0auth/message';
+import {
+  encryptMessage,
+  getDecryptedMessageFromLocalStorage,
+  getRawKeyFromLocalStorageIfPresentElseGenerate,
```

Comment:
Isn't too long the name of the function?
How about changing like this?
```suggestion
  getDecryptedMsg,
  getGeneratedRawKey, 
```
Because this function generates if there is no raw key and returns it (if the key was present, that is also generated before.), so its name would be more clear and looks good with `getGeneratedKey`. 
In my opinion, the sentence`If~Else~` doesn't need to put in the name of the function.
And, `FromLocalStorage` too.

If you want to apply similar functions to `IndexedDB` and `ChromeExtension` when they're implemented, I think it's better to implement it like below.
```js
function getKeyFromLocalStorage(): string | null {
...
}

function getRawKeyFromLocalStorageIfPresentElseGenerate(...): string {
...
}

// ... all other functions
export const getSpecificFunction = {
  'GET_KEY': {
    'INDEXED_DB': function () {},
    'LOCAL_STORAGE': function () {...},
    ...
  },
  'SET_KEY': {
  ...
  },
...
}
```

I think the name change is a recommendation, but the format change below is just a suggestion.